### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by brentleyjones**
@yonaskolb You can manually push these (or a version of these) as well. Just noticed we missed two entries.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
 - Added ability to specify `copyFiles` build phase for sources [345](https://github.com/yonaskolb/XcodeGen/pull/345) @brentleyjones
+- Added ability to specify a `minimumXcodeGenVersion` [349](https://github.com/yonaskolb/XcodeGen/pull/349) @brentleyjones
 
 #### Fixed
 - Sort files using localizedStandardCompare [341](https://github.com/yonaskolb/XcodeGen/pull/341) @rohitpal440
 - Use the latest `xcdatamodel` when sorted by version [341](https://github.com/yonaskolb/XcodeGen/pull/341) @rohitpal440
 - Fixed compiler flags being set on non source files in mixed build phase target sources [347](https://github.com/yonaskolb/XcodeGen/pull/347) @brentleyjones
 - Fixed `options.xcodeVersion` not being parsed [348](https://github.com/yonaskolb/XcodeGen/pull/348) @brentleyjones
 - Fixed non-application targets using `carthage copy-frameworks` [361](https://github.com/yonaskolb/XcodeGen/pull/361) @brentleyjones
+- Set `xcdatamodel` based on `xccurrentversion` if available [364](https://github.com/yonaskolb/XcodeGen/pull/364) @rpassis
```
Comment (yonaskolb) [inline CHANGELOG.md]: I'd put this under added

**Comment by yonaskolb** [comment]
Thanks! Yeah I usually update these manually periodically or before a release. I don't require it from PR authors due to the code conflicts it always causes

**Comment by brentleyjones** [comment]
We can use `merge union` in `.gitattributes` to fix the merge conflicts: https://about.gitlab.com/2015/02/10/gitlab-reduced-merge-conflicts-by-90-percent-with-changelog-placeholders/

I'll let you update this :smile:.

**Comment by yonaskolb** [comment]
Nice tip! Added in https://github.com/yonaskolb/XcodeGen/commit/b918a1e76b4b98883de2a8ba99a5dbce435764f8

**Pull request closed without merging**